### DISCUSSION TITLE
How to get Nickname(displayName/globalName)?

### DISCUSSION THREAD

**Discussion by ghost** [bwmarrin/discordgo#1444] [Q&A]
In handler function, I'm able to get member joined to channel but it only has either username which is not what is displayed in channel or "Nick" field but it's always empty.
`func joined(s *discordgo.Session, m *discordgo.VoiceStateUpdate) {
	//g, _ := s.State.Guild(m.GuildID)
	//c, _ := s.State.Channel(m.ChannelID)
	mem, _ := s.State.Member(m.GuildID, m.UserID)
	log.Printf("User %s joined channel ...", mem.Nick)
}`

**Comment by Earlopain** [comment]
Support for global names has not been released yet, you will need to run from master if you need access to that right now. pr #1426 aims to make the display name easily accessible without doing a bunch of checks yourself
